src/app/cube/about-us/components/about-clark/about-clark.component.html

```diff
@@ -3,21 +3,40 @@
+        <p>
+          CLARK hosts the largest compilation of free, high-value, high-impact <span class="blue_text">CYBERSECURITY</span> 
+          curriculum created by top researchers that has been peer-reviewed by accredited instructional designers and 
+          subject matter experts. Instructors looking to teach new cybersecurity <span class="blue_text" >COURSES</span> or 
+          expand and improve their own instruction, can find ready-to-use modular content including videos, 
+          labs, and slides that can be reused and modified under Creative Commons 4.0.
+        </p>
       </div>
       <img class = "fs-img" src="../../../../../assets/images/collections/secinj.png" alt="Code Responsibly">
     </div>
       
     <div class="second-section">
       <img class = "ss-img" src="../../../../../assets/images/module.JPG" alt="Module Example">
       <div class="ss-text-container">
-        <p>CLARK hosts the largest compilation of high-value, high-impact <span class="blue_text">CYBERSECURITY</span> curriculum created by top researchers, peer-reviewed by instructional designers and subject matter experts. Instructors looking to teach new cybersecurity <span class="blue_text" >COURSES</span> or expand and improve their own instruction, can find ready-to-use modular content, including videos, labs, and slides that can be modified under the Creative Commons license.</p>
+        <p>
+          CLARK supports the model for <span class="blue_text" >BUILDING</span>  
+          modularized curriculum, the digital library system to <span class="blue_text" >BROWSE</span> 
+          free curriculum, and unique <span class="blue_text" >COLLECTIONS</span>
+          to categorize distinct groups of curriculum. CLARK provides cybersecurity educators the tools needed
+          to better prepare and equip the <span class="blue_text">NEXT WAVE</span> of cybersecurity professionals and researchers of tomorrow. 
+        </p>
       </div>
     </div>
 
     <div class="third-section">
       <div class="ts-text-container">
-        <p>The CLARK system, hosted at <a href="www.clark.center" aria-label="Clickable Home page link">www.clark.center</a> , was created as a high-availability, relevant, and scalable repository for curricular resources in the cybersecurity education community. With several thousand users, CLARK is gaining traction as the goto source for free peer-reviewed cybersecurity curriculum. Designed based on lessons-learned from research on digital libraries and learning sciences, CLARK is easy to use by both contributors and consumers.</p>
```

Comment:
There is a typo in the sentence "CLARK is establishing itself as the _goto_ source for free peer-reviewed cybersecurity curriculum." The sentence should read "CLARK is establishing itself as the _go to_ source for free peer-reviewed cybersecurity curriculum."